docs/common-issues.md

```diff
@@ -10,6 +10,48 @@ This could be due to a regression, or the feature is not yet implemented. If you
 certain query syntax, make sure it is implemented first before raising an issue. Here is the list of
 selectors we currently support: https://github.com/airbnb/enzyme/blob/master/docs/api/selector.md
 
+### Nested component may not be updated after wrapper updated
+
+Assume we have simple component with `<input />` and `<button>`. Clicking `<button>` updates `<input>`'s value. But test
+```jsx
+const input = wrapper.find('input');
+const button = wrapper.find('button');
+button.props().onClick();
+expect(input.props().value).toEqual('test');
+```
+fails. While
+```jsx
+const input = wrapper.find('input');
+const button = wrapper.find('button');
+button.props().onClick();
+expect(wrapper.find('input').props().value).toEqual('test');
+```
+passes.
+
+This is related to both `mount()` and `shallow()`, `find()` and `findWhere` methods.
```

Comment:
It relates to any method that can produce a new wrapper; filter, filterWhere, find, findWhere, dive, mount, shallow, etc. Can we find a way to rephrase this so it's not an explicit list?

Reply:
Rephrased. Not really sure if it should be `wrapper` or `wrappers` when goes to list/collection.

Reply:
It's a single wrapper, around 0, 1, or more things, so `wrapper` is correct.

Reply:
is there anything else I should fix?